controller-server/src/test/java/com/yahoo/vespa/hosted/controller/restapi/application/ApplicationApiTest.java

```diff
@@ -661,17 +661,21 @@ public void testApplicationApi() {
                                       .data(streamer),
                               "{\"message\":\"Application package version: 1.0.3-commit1, source revision of repository 'repository1', branch 'master' with commit 'commit1', by a@b, built against 6.1 at 1970-01-01T00:00:01Z\"}");
 
-        // Sixth attempt has a multi-instance deployment spec, and fails.
+        // Sixth attempt has a multi-instance deployment spec, and is accepted.
         ApplicationPackage multiInstanceSpec = new ApplicationPackageBuilder()
                 .instances("instance1,instance2")
+                .systemTest()
+                .stagingTest()
                 .environment(Environment.prod)
                 .region("us-central-1")
                 .parallel("us-west-1", "us-east-3")
+                .endpoint("default", "foo", "us-central-1", "us-west-1", "us-east-3")
```

Comment:
Is there a test that verifies endpoint configuration combined with multiple instances?

Reply:
What calls are made against the rotation repository?

Reply:
We haven't really supported multiple instances in the deployment spec before, so no. I'll add one. Can you think of other places such tests are needed?

Reply:
Ah, endpoints.